settings.gradle.kts

```diff
@@ -0,0 +1,62 @@
+pluginManagement {
+  repositories {
+    mavenCentral()
+    gradlePluginPortal()
+    maven {
```

Comment:
In `build-logic/settings.gradle.kts` the order of repositories is different. It should be consistent. I would suggest this order as it will not break the build for someone from the community